features/vcr_cassettes/Dashboard/Performing_site_autodiscovery.yml

```diff
@@ -152,4 +152,1335 @@ http_interactions:
         }
     http_version: 
   recorded_at: Thu, 22 Feb 2018 07:39:53 GMT
+- request:
```

Comment:
This recording should not be updated as part of this commit. When the Rails 5 changes are complete, we should re-run all the specs locally in case any cassettes need to be updated at that point.

Reply:
ah that one slipped in. Good catch!